tools/packages/fink/build.properties

```diff
@@ -1,45 +0,0 @@
-###############################################################################
```

Comment:
we should delete the fink build altogether, I doubt it works, and it's definitely unmaintained